services/payments/datastore.go

```diff
@@ -130,16 +129,6 @@ func (q *QLDBDatastore) GetPaymentStateHistory(ctx context.Context, documentID s
 			return nil, &QLDBTransitionHistoryNotFoundError{}
 		}
 
-		merkleValid, err := revisionValidInTree(ctx, q.sdkClient, &latestHistoryItem)
```

Comment:
We should probably remove the function itself as well. I don't think we plan to use it.